omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/api/Deletable.java

```diff
@@ -28,4 +28,16 @@
 	 */
 	void delete(String uuid, String reason, RequestContext context) throws ResponseException;
 	
+	/**
+	 * Undeletes the specified resource, which in the OpenMRS context means either unvoiding or
+	 * unretiring it
+	 * 
+	 * @param uuid Uuid to the delegate to unvoid/unretire
```

Comment:
What does "delegate" mean?

Reply:
Delegate in my thinking as I have seen it being used with respect to this module, is a domain object.

Reply:
What has that got to do with the API consumer?

Reply:
I think to make this doc safe, I'l replace "delegate" with "resource" instead.